dali/kernels/signal/window/extract_windows_cpu.cc

```diff
@@ -47,8 +46,11 @@ KernelRequirements ExtractWindowsCpu<OutputType, InputType, Dims>::Setup(
 
   window_length_ = args.window_length > 0 ? args.window_length : 1;
   window_step_ = args.window_step > 0 ? args.window_step : 1;
-  window_center_offset_ = args.window_center < 0 ? window_length_ / 2 : args.window_center;
-  reflect_pad_ = args.reflect_pad;
+  padding_ = args.padding;
+  if (padding_ != Padding::None)
+    window_center_offset_ = args.window_center < 0 ? window_length_ / 2 : args.window_center;
```

Comment:
What if window_center  is < than window_length_ / 2?
Do we pad at the beginning as well?